components/style/values/specified/length.rs

```diff
@@ -206,10 +193,11 @@ impl FontRelativeLength {
     }
 }
 
-#[derive(Clone, Copy, Debug, MallocSizeOf, PartialEq)]
 /// A viewport-relative length.
 ///
 /// <https://drafts.csswg.org/css-values/#viewport-relative-lengths>
+#[derive(Clone, Copy, Debug, MallocSizeOf, PartialEq, ToCss)]
+#[css(dimension)]
```

Comment:
Please make it such that this attribute go on the dimension variants, not on the type itself. CSS doesn't say that things that use dimensions should always use dimensions.

Also, document it in `ToCss`.